docs/metrics/customizing-the-sdk/Program.cs

```diff
@@ -0,0 +1,61 @@
+// <copyright file="Program.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Diagnostics.Metrics;
+using OpenTelemetry;
+using OpenTelemetry.Metrics;
+
+public class Program
+{
+    private static readonly Meter MyMeter = new Meter("MyCompany.MyProduct.MyLibrary", "1.0");
+
+    public static void Main(string[] args)
+    {
+        using var meterProvider = Sdk.CreateMeterProviderBuilder()
+            .AddSource(MyMeter.Name)
+            .AddView(instrumentName: "MyCounter") // MyCounter will be reported with defaults. This is done to ensure that any other wildcard Views for the same isntrument does not affect this instrument.
+            .AddView(instrumentName: "MyHistogram") // MyHistogram will be aggregated using defaults.
+            .AddView(instrumentName: "MyCounterDrop", aggregation: Aggregation.None) // The intention is to drop MyCounterDrop instrument. Due to the wildcard "my", this gets pickup.. TODO: find the right expectation.
+            .AddView(name: "MyHistogramCustom", instrumentName: "MyHistogram", histogramBounds: new double[] { 10, 20 }) // MyHistogram will be aggregated using custom bounds and will be outputted as "MyHistogramCustom"
+            .AddView(instrumentName: "My", tagKeys: new string[] { "tag1" }) // All instruments whose name starts with My, will be aggregated with a single tag - tag1. MyCounter, MyHistogram will get excluded from this, as their name is already taken.
```

Comment:
Is this the wildcard "my" mentioned above, is it missing a `*`?

Reply:
just used StartsWith now. It'll need to changed with a proper regex.